numpy/warnings.pyi

```diff
@@ -0,0 +1,4 @@
+from typing import Any
```

Comment:
This one is just the built in `warnings` module IIRC, so we don't want to include it?

Reply:
You're absolutely right, fixed in https://github.com/numpy/numpy/pull/17104/commits/e91e31eab3f6a4af1498635c6c80d4ef4263e1d9.